<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingWindow</class>
 <widget class="QDialog" name="TestingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="QuestionTextBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>501</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="option1Button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>16</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="option2Button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="option3Button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="option4Button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="option1TextBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="option2TextBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="option3TextBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="option4TextBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nextQuestion">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="option5TextBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="option5Button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
